src/chunk.h

```diff
@@ -32,6 +32,7 @@ Chunk * NewChunk(size_t sampleCount);
 void FreeChunk(Chunk *chunk);
 
 // 0 for failure, 1 for succss
+Sample *ChunkGetSample(Chunk *chunk, int index);
```

Comment:
Chunk functions should be generic for all chunk types, currently we have uncompressed chunks only so its possible to use an index, but with compressed chunk its different. This is why `ChunkGetSample` is not exported to `chunk.h`.

Reply:
I don't sure i understand...
You want me to support something that isn't yet implemented?
even if so, how am I suppose to do that without knowing anything about the compressed chunks possible implementation?